web_app/calendar/static/pages/calendar_page.html

```diff
@@ -4,10 +4,10 @@
     <style>
         table, th, td {{{{border: 1px solid black; collapse;}}}}
         a {{{{
-            text-decoration: none; color: black;
+            text-decoration: none; color: black; display: block;
         }}}}
         td.icon-cell {{{{
-            width: 22px;
+            width: 26px;
```

Comment:
Better use properties in % than in fixed values

Reply:
It is page style ) we will make it more beautiful later